sandbox/gas/tube/go

```diff
@@ -195,7 +195,7 @@ def tube_base(casename=None,
     # Report information while calculating.
     cse.defer(gas.ProgressHook, linewidth=ssteps / psteps, psteps=psteps)
     cse.defer(gas.CflHook, fullstop=False, cflmax=10.0, psteps=ssteps)
-    cse.defer(gas.MeshInfoHook, psteps=ssteps)
+    cse.defer(gas.MeshInfoHook, psteps=ssteps, show_bclist=True)
```

Comment:
Only for verbose information to catch and verify the fix. It is not related to fix the compatibility directly.